src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Configuration/IConfiguration.cs

```diff
@@ -15,6 +15,8 @@ public interface IConfiguration
         string AgentEnabledAt { get; }
 
         bool ServerlessModeEnabled { get; }
+        string ServerlessFunctionName { get; }
+        string ServerlessFunctionVersion { get; }
```

Comment:
We might need to update the tests with the large json blobs to keep the test coverage numbers up with these additions.